km/km_guest_asmcode.s

```diff
@@ -89,7 +89,7 @@ intr_hand CP, 21
 /*
  * Table used by KM to build IDT entries.
  */
-    .section .km_guest_data, "da", @progbits
+    .section .km_guest_data, "dwa", @progbits
```

Comment:
do you want to change permissions to write on trap table?

Reply:
I'm not sure when this went in or why. @gnode1 ?

Reply:
static pie link was complaining otherwise